plugins/win-capture/game-capture.c

```diff
@@ -1044,7 +1073,8 @@ static void setup_window(struct game_capture *gc, HWND window)
 	 * (such as steam) need a little bit of time to load.  ultimately this
 	 * helps prevent crashes */
 	if (gc->wait_for_target_startup) {
-		gc->retry_interval = 3.0f;
+		gc->retry_interval = 3.0f*
```

Comment:
Style: Typically, you need to surround math operators by a single space on each side. Don't leave a trailing space at the end of the line.